sparse/etc/systemd/system/sensorfwd.service.d/90-boot-order.conf

```diff
@@ -1,3 +1,3 @@
 [Unit]
-Requires=systemd-modules-load.service
-ConditionPathExists=/dev/iio:device1
+Requires=systemd-modules-load.service sys-devices-0044:8086:22D8.0001-HID\x2dSENSOR\x2d200083.6.auto-iio:device4.device
```

Comment:
We can avoid checking for `systemd-modules-load.service` now (but I'm OK leaving it there)